.gitlab-ci.yml

```diff
@@ -284,7 +284,7 @@ docker-boot:
   script:
     - dev/tools/check-cachekey.sh
     - docker login -u gitlab-ci-token -p "$CI_JOB_TOKEN" "$CI_REGISTRY"
-    - cd dev/ci/docker/bionic_coq/
+    - cd dev/ci/docker/
```

Comment:
The reason we choose to reflect the image name in the directory / variables is that at some point we thought it could be the case that we had different images for different OS versions.

I'm OK with this change if you find it more readable.

Reply:
Yes, that's how I understood it. I figured that, if after 5 years, we still had only a single Docker image, then this structure was not worth keeping. We can always revive it later if we want.